README.md

```diff
@@ -36,6 +36,10 @@ We enforce our coding standards with automated tooling ("linters"), that are usu
 
 Refer to [Mofo Style](https://github.com/MozillaFoundation/mofo-style) for more info.
 
+## License
+
+Make sure every repo contains a [MPL 2.0](https://www.mozilla.org/en-US/MPL/) `LICENSE` file. This `LICENSE` file can be added by selecting "Mozilla Public License 2.0" when you create a new repo on GitHub.
```

Comment:
small linguistic nit: "an MPL 2.0 LICENSE file" (because MPL is an acronym starting with a vowel sound)